packages/amplify-e2e-tests/src/__tests__/graphql-v2/searchable-datastore.test.ts

```diff
@@ -27,7 +31,8 @@ describe('transformer model searchable migration test', () => {
   });
 
   afterEach(async () => {
-    await deleteProject(projRoot);
+    const newCreds = refreshCredentials();
```

Comment:
Can we add a codebuild ci check here? The credential refresh is only needed in CodeBuild CI and not needed if running locally or in CircleCI. The current env variables used are `process.env.CI && process.env.CODEBUILD`

Reply:
Added an early return inside the refresh method, that should take care of this.